apps/src/applab/designElements/PropertyRow.jsx

```diff
@@ -15,6 +17,10 @@ var PropertyRow = React.createClass({
     };
   },
 
+  componentWillReceiveProps: function (newProps) {
```

Comment:
I don't remember exactly, but I think it might be that the HTML Element changes, we call rerender on the DesignProperties component, but the row didn't update because its state remained unchanged.

One time this will happen is for the image element, where on load we change the element width/height and ask for a rerender.